File: src/Java/Task/Task.java
package Java.Task;

import java.sql.*;

public class Task {
    String name;
    int duration;
    int priority;
    Date dueDate;

    Task(String name, int duration, int priority, Date dueDate) {
        if (name != null){
            this.name = name;
        } else {
            this.name = "placeholder"; //TODO ska va ett error här, man får inte ha en task utan namn
        }
        if (duration != 0){
            this.duration = duration;
        } else{
            this.duration = 0; //för senare calcs så får duration och priority aldrig va null
        }
        if (priority > 0 || priority < 6){
            this.priority = priority;
        } else {
            this.priority = 1;
        }

    }
}

File: src/components/Task.js
import "./Task.css";

function Task(props) {

    return (
        <div className="task-item"> 
        <h1 className="task-name"> Task Name </h1>
        <h2 className="task-priority"> Priority </h2>
        <h2 className="task-duration"> Duration </h2>

        </div>
    )
}

export default Task;